admin/data/sql/migrations/20191112222703.do.add-aliases-pupil-table.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE mtc_admin.pupil ADD foreNameAlias [NVARCHAR](max) NULL
```

Comment:
MAX can be 2GB which seems a little high for an alias.  Perhaps both of these aliases should be limited to 128 chars?  How much of the 2GB would we actually display?

@GuyHarwood - thoughts?

Reply:
@jon-shipley we should change then middleNames as well as this follows the same pattern

Reply:
Good idea.

Reply:
agreed

Reply:
done